docs/manuals/run-deploy/deploy/databricks/index.md

```diff
@@ -0,0 +1,124 @@
+# Deploy on Databricks
+
+Databricks is a standard platform for data scientists and big data engineers. It can provide a web-based laptop connected to your data source within your cloud provider. Nevertheless, Databricks does not provide a direct path to developing a web application. This documentation will show you how to adapt your infrastructure so that you can create a Taipy application in your Databricks.
```

Comment:
"web-based laptop"??

"...application in your Databricks"
=>
"application in your Databricks environment"

"cloud provider" -> "Cloud provider" (? not too sure... and if yes, everywhere in this document)